goverge/test/test_main.py

```diff
@@ -210,6 +218,23 @@ def test_race(self):
             'short': False,
             'tag': None,
             'test_path': None,
+            'threads': 4,
+            'xml': False,
+            'xml_dir': 'xml_reports/'
+        }
+        self.assertEquals(expected, vars(args))
+
+    def test_threads(self):
+        args = main._parse_args(["--threads=10"])
+        expected = {
+            'godep': False,
+            'html': False,
+            'project_import': None,
+            "race": False,
+            'short': False,
+            'tag': None,
+            'test_path': None,
+            'threads': '10',
```

Comment:
That's a good point, I guess I just followed what I saw somewhere else. I will change this test and make a follow up PR to refactor all the other tests to match.